tests/test_pipelines_ner.py

```diff
@@ -105,53 +107,66 @@ def _test_pipeline(self, nlp: Pipeline):
+    #         pipeline_running_kwargs = {"offset_mapping"}  # Additional kwargs to run the pipeline with
+
     @require_tf
     def test_tf_defaults(self):
         for model_name in self.small_models:
-            nlp = pipeline(task="ner", model=model_name, tokenizer=model_name, framework="tf")
+            tokenizer = AutoTokenizer.from_pretrained(model_name, use_fast=True)
+            nlp = pipeline(task="ner", model=model_name, tokenizer=tokenizer, framework="tf")
         self._test_pipeline(nlp)
 
     @require_tf
     def test_tf_small(self):
         for model_name in self.small_models:
+            tokenizer = AutoTokenizer.from_pretrained(model_name, use_fast=True)
             nlp = pipeline(
                 task="ner",
                 model=model_name,
-                tokenizer=model_name,
+                tokenizer=tokenizer,
                 framework="tf",
                 grouped_entities=True,
                 ignore_subwords=True,
             )
             self._test_pipeline(nlp)
-        for model_name in self.small_models:
-            nlp = pipeline(
-                task="ner",
-                model=model_name,
-                tokenizer=model_name,
-                framework="tf",
-                grouped_entities=True,
-                ignore_subwords=False,
-            )
-            self._test_pipeline(nlp)
+
+    #         for model_name in self.small_models:
+    #             tokenizer = AutoTokenizer.from_pretrained(model_name, use_fast=True)
+    #             nlp = pipeline(
+    #                 task="ner",
+    #                 model=model_name,
+    #                 tokenizer=tokenizer,
+    #                 framework="tf",
+    #                 grouped_entities=True,
+    #                 ignore_subwords=False,
+    #             )
+    #             self._test_pipeline(nlp)
 
     @require_torch
     def test_pt_defaults(self):
         for model_name in self.small_models:
-            nlp = pipeline(task="ner", model=model_name, tokenizer=model_name)
+            tokenizer = AutoTokenizer.from_pretrained(model_name, use_fast=True)
+            nlp = pipeline(task="ner", model=model_name, tokenizer=tokenizer)
             self._test_pipeline(nlp)
 
     @require_torch
     def test_torch_small(self):
         for model_name in self.small_models:
+            tokenizer = AutoTokenizer.from_pretrained(model_name, use_fast=True)
             nlp = pipeline(
-                task="ner", model=model_name, tokenizer=model_name, grouped_entities=True, ignore_subwords=True
-            )
-            self._test_pipeline(nlp)
-        for model_name in self.small_models:
-            nlp = pipeline(
-                task="ner", model=model_name, tokenizer=model_name, grouped_entities=True, ignore_subwords=False
+                task="ner", model=model_name, tokenizer=tokenizer, grouped_entities=True, ignore_subwords=True
```

Comment:
Don't know why the tests are failing, I get normal results when running them outside of the test suite 😕 ?